tests/src/python/test_qgsprojectutils.py

```diff
@@ -115,14 +120,23 @@ def test_updateLayerPath(self):
         self.assertFalse(QgsProjectUtils.updateLayerPath(p, "aaa", "bbb"))
 
         # replace shapefile path
+
+        ## copy files to temp directory
+        tempDir = tempfile.TemporaryDirectory()
+        tempDirPath = Path(tempDir.name).as_posix()
```

Comment:
no need to call `as_posix()`. `shutil.copy` can handle `Path` objects

Reply:
yes! It is a test remain :-1: